src/index.ts

```diff
@@ -241,11 +246,27 @@ function initUpdater(opts: ReturnType<typeof validateInput>) {
     );
   }
 
-  // check for updates right away and keep checking later
-  autoUpdater.checkForUpdates();
-  setInterval(() => {
+  if (autoCheck) {
```

Comment:
```suggestion
  if (autoCheck === true) {
```

I wonder if we want to be stricter about actually passing `true` rather than something truthy.

Reply:
https://github.com/electron/update-electron-app/blob/c5f1d43e52914d57b95e2239c8cec2cd6815679d/src/index.ts#L216

I follow this, so I didn't do strict equal
I agree with you, it might need stricter